.github/workflows/main.yml

```diff
@@ -0,0 +1,16 @@
+on: [push]
```

Comment:
This is just for testing it, right?

Reply:
Yes! It is a way to run the action even though it hasn't been published yet